civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -569,6 +569,32 @@ function civihr_employee_portal_init() {
     _rebuild_view('appraisal');
     _rebuild_view('tasks');
     _rebuild_view('documents');
+
+    $extensionPath = CRM_Extension_System::singleton()->getMapper()->keyToUrl('uk.co.compucorp.civicrm.tasksassignments');
+
+    $allSettings = array();
+    $taSettings = civicrm_api3('TASettings', 'get');
+    foreach ($taSettings['values'] as $key => $value) {
+      $allSettings[$key] = $value['value'];
+    }
+    if (!empty($allSettings['days_to_create_a_document_clone'])) {
+      $allSettings['days_to_create_a_document_clone'] = (int)$allSettings['days_to_create_a_document_clone'];
+    }
+
+    $settings = [
```

Comment:
These settings are required to bootstrap T&A app, actually, I have taken these settings form T&A in `CRM/Tasksassignments/Page/Tasksassignments.php` file.